src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -291,10 +296,10 @@ def parse_performance_features(log_lines, strategies, arguments):
 
   # new_cov_* is a reliable metric when corpus subset strategy is not used.
   if not stats['strategy_corpus_subset']:
-    if 'initial_edge_coverage' in stats and 'edge_coverage' in stats:
```

Comment:
This was incorrect as these metrics were inited to 0. so, this was always true, instead we should check if end stats metrics is non-zero.

Reply:
Why do we need these `if`s at all? If the above code works shouldn't feature_coverage always be 0 if initial_feature_coverage is zero? Same for edge. Maybe change to an assert.

Reply:
Done